releases/codefresh.yaml

```
@@ -0,0 +1,259 @@
+repositories:
+# Codefresh on-prem repository
+- name: "codefresh"
+  url: 'http://charts.codefresh.io/{{ env "CHANNEL" | default "dev" }}'
+# Incubator repo of official helm charts
+- name: "incubator"
+  url: "https://kubernetes-charts-incubator.storage.googleapis.com/"
+
+releases:
+
+#######################################################################################
+## Codefresh On-Prem                                                                 ##
+#######################################################################################
+
+#
+# References:
+#   - https://github.com/codefresh-io/onprem
+#
+- name: "cf"
+  namespace: "codefresh"
+  labels:
+    chart: "codefresh"
+    component: "codefresh"
+    namespace: "codefresh"
+    vendor: "codefresh"
+    default: "false"
+  chart: "codefresh/codefresh"
+  version: "0.7.9"
+  wait: true
+  values:
+  - global:
+      ### Instantiate databases with seed data. Usually used in dynamic and on-prem environments.
+      seedJobs: {{ env "CODEFRESH_SEED_JOBS_ENABLED" | default "false" }}
+      certsJobs: {{ env "CODEFRESH_SEED_JOBS_ENABLED" | default "false" }}
+      appProtocol: https
+      ### Codefresh App domain name
+      appUrl: {{ env "CODEFRESH_HOST" }}
+{{- if eq (env "CODEFRESH_SELF_HOSTED_POSTGRESQL_ENABLED" | default "true") "false" }}
+      postgresDatabase: {{ env "AURORA_POSTGRES_DATABASE_NAME" }}
```

```suggestion
      postgresDatabase: {{ env "CODEFRESH_POSTGRES_DATABASE_NAME" }}
```

`CODEFRESH_` for consistency with the other vars.
not `AURORA_` because it could be any other Postgres database, e.g. RDS

---

based on values what we have in ssm param store created with terraform